app/templates/flatpages/privacy-policy.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block head %}
+<title>Privacy Policy | HackItForward</title>
+{% endblock %}
+
+{% block body %}
+<space class="large"></space>
+<div class="content" style="margin-bottom: 90px;">
+  <h1 style="text-align: center; margin-bottom: 40px;">Privacy Policy</h1>
+  <ul>
+    <li>HackItForward stores a small amount of data such as your username, your profile picture and other user-generated data on the platform when you create an account.</li>
+    <li>Google and Github OAuth are used to facilitate the registration process.</li>
+    <li>All data is stored securely on a cloud server hosted with Digital Ocean. It will not be shared with any third party. </li>
+    <li>You can request to have all of your data deleted from the system at any time by emailing info@kurius.ca</li>
```

Comment:
Mention how you can request all of your data -- this is something I believe is required under privacy regulations (I am not a lawyer).